core/assets/bundles/bundle_fr.properties

```
@@ -1106,11 +1164,12 @@ block.phase-conduit.description = Tuyau très avancé permettant le transport de
 block.power-node.description = Transmet l'énergie aux transmetteurs énergétiques connectés. Le transmetteur recevra de l'énergie ou la transmettra à n'importe quel bâtiment adjacent.
 block.power-node-large.description = Possède un rayon plus grand que le transmetteur énergétique standard, connectant d'autant plus de consommateurs ou transmetteurs d'énergie.
 block.surge-tower.description = Un transmetteur énergétique de très grande portée mais avec moins de connections disponibles.
+block.diode.description = La batterie ne peut circuler dans ce bloc que dans un sens, mais uniquement si l’autre côté a moins d’énergie stockée.
 block.battery.description = Stocke l'énergie quand elle est en abondance et la redistribue si il y a un deficit d'énergie dans la limite des réserves disponibles.
 block.battery-large.description = Stocke bien plus d'énergie qu'une batterie normale.
-block.combustion-generator.description = Génère de l'énergie en brûlant du charbon ou des matériaux inflammables.
-block.thermal-generator.description = Génère une grande quantité d'énergie à partir de zone de chaleur .
-block.turbine-generator.description = Plus efficace qu'un générateur à combustion, mais requiert de l'eau .
+block.combustion-generator.description = Génère de l'énergie en brûlant du charbon ou autres matériaux inflammables.
```

`d'autres` because `du charbon`.
```suggestion
block.combustion-generator.description = Génère de l'énergie en brûlant du charbon ou d'autres matériaux inflammables.
```